SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1374,6 +1374,12 @@ void ActorSystem::ReceiveActor(Worker_EntityId EntityId)
 	}
 
 	ApplyFullState(EntityId, *Channel, *EntityActor);
+
+	const UNetConnection* ActorNetConnection = EntityActor->GetNetConnection();
+	if (IsValid(ActorNetConnection) && NetDriver->ServerConnection == ActorNetConnection)
+	{
+		NetDriver->OwnershipCompletenessHandler->AddPlayerEntity(EntityId);
```

Comment:
Isn't "OwnershipCompletenessHandler" optional?

Reply:
It's only optional in the sense that we can't create it on an Unreal object (the NetDriver) during the latter's construction; it is set in `USpatialNetDriver::InitBase` and is never reset.

Reply:
That's a fair point, I'll update the code as you're suggesting.